catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -356,6 +356,14 @@
         </bundle>
     </feature>
 
+    <feature name="catalog-core-validator" install="manual" version="${project.version}"
```

Comment:
This feature includes the existing metacard validation plugin as well. Do we want that installed by default?